Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py

```diff
@@ -0,0 +1,131 @@
+from CommonServerPython import *
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def __init__(self, base_url, verify=True, proxy=False, ok_codes=(), headers=None, auth=None, token=None):
+        super().__init__(base_url, verify, proxy, ok_codes, headers, auth)
+        self.token = token
+
+    def get_file_report(self, file_hash: str):
+        return self._http_request(
+            'POST',
+            url_suffix='/get/report',
+            params={'apikey': self.token, 'format': 'pdf', 'hash': file_hash},
+            resp_type='response',
+            ok_codes=(200, 404),
+        )
+
+
+''' HELPER FUNCTIONS '''
+
+
+def hash_args_handler(sha256=None, md5=None):
+    inputs = argToList(sha256) if sha256 else argToList(md5)
+    for element in inputs:
+        if sha256Regex.match(element) or md5Regex.match(element):
+            continue
+        raise Exception('Invalid hash. Only SHA256 and MD5 are supported.')
+
+    return inputs
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client):
+    try:
+        wildfire_hash_example = 'dca86121cc7427e375fd24fe5871d727'  # guardrails-disable-line
```

Comment:
The hash example was taken from https://docs.paloaltonetworks.com/wildfire/u-v/wildfire-api/get-started-with-the-wildfire-api/make-your-first-wildfire-api-call.html#id67bf31b8-9a8f-41e9-a98f-c632571c2255